site/assets/scss/_variables.scss

```diff
@@ -39,7 +39,7 @@ $bd-callout-variants: info, warning, danger !default;
   // Boosted mod: no --bd-sidebar-link-bg
   // Boosted mod: no --bd-callout-link
   // Boosted mod: no --bd-callout-code-color
-  --bd-pre-bg: #{adjust-color($gray-900, $lightness: -2.5%)}; // stylelint-disable-line scss/at-function-named-arguments
```

Comment:
`var(--bd-pre-bg)` is still used two times in this branch in __component-examples.scss_.

Reply:
Yes, but we don't need to overwrite it in dark mode. The light mode definition is already fine to me. However, it changes the background-color of the code examples. Are you fine with this change ?